problems/34.Counting Words With a Given Prefix.gs

```diff
@@ -0,0 +1,15 @@
+function prefixCount(words: string[], pref: string): number {
+    let cnt = 0
+
+    for (let i = 0; i < words.length; i++) {
+        if (words[i].substring(0, pref.length) == pref) {
```

Comment:
at last use proper function if you are going to use builtin...
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith
And you will get much needed performance...